tasks/deploy.js

```diff
@@ -67,38 +67,38 @@ task(
         saleFee
       );
 
-      // Deploy EasyAuction
-      const EasyAuction = hre.artifacts.require(
-        "EasyAuction"
+      // Deploy FairSale
+      const FairSale = hre.artifacts.require(
+        "FairSale"
       );
-      const easyAuction = await EasyAuction.new();
+      const fairSale = await FairSale.new();
 
        // Deploy FixedPriceSale
        const FixedPriceSale = hre.artifacts.require(
         "FixedPriceSale"
       );
       const fixedPriceSale = await FixedPriceSale.new();
 
-      // Register EasyAuction & FixedPriceSale in SaleLauncher
-      const saleLaunch1 = await saleLauncher.addTemplate(easyAuction.address);
+      // Register FairSale & FixedPriceSale in SaleLauncher
+      const saleLaunch1 = await saleLauncher.addTemplate(fairSale.address);
       const saleLaunch2 = await saleLauncher.addTemplate(fixedPriceSale.address);
 
       const sale1 = saleLaunch1.receipt.logs[0].args.templateId;
       const sale2 = saleLaunch2.receipt.logs[0].args.templateId;
 
-      // Deploy EasyAuctionTemplate
-      const EasyAuctionTemplate = hre.artifacts.require(
-        "EasyAuctionTemplate"
+      // Deploy FairSaleTemplate
+      const FairSaleTemplate = hre.artifacts.require(
+        "FairSaleTemplate"
       );
       
-      const easyAuctionTemplate = await EasyAuctionTemplate.new(
+      const fairSaleTemplate = await FairSaleTemplate.new(
          weth,
          saleLauncher.address,
          sale1
       );
       
```

Comment:
Having vars only to differ with capital letters and one without.  I would say this is risky and can be confusing. Coding style wise: we never use capital letters for vars otherwise? 

Can't we use fairSale all the time?

Reply:
Hey @martinkrung, the naming convention idea here is to reference contracts with their exact Solidity contract name (Capitalized), while contract instances are written in camelCase.